### PR TITLE
Hero section turned to 800 px tall for 1440p

### LINKED ISSUE #1680: [Community] Hero section too small
**Description**
Hero section needs to be a bit taller according to the design. (800 px tall for 1440p wide screens)

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: Anita-ihuman <charlesanita403@gmail.com>

**Description**
I changed the Hero section to 800 px tall for 1440p wide screens
This PR fixes #1680 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b317b25555a5aa18ebf4d6e899808d74e46c4e6f at: https://605f2131b72fdf962345f8b9--layer5ng.netlify.app

File: src/sections/Community/community.style.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const CommunitySectionWrapper = styled.div`
     margin-bottom: 3.25rem;
     .community-header{
         color: white;
-        height: 25rem;
+        height: 50rem;
```
Comment (Nikhil-Ladha) [inline src/sections/Community/community.style.js:8]: It's way too long, and distorts the image.
Make it 35rem, that would be fine.

**Comment by l5io** [comment]
🚀 Preview for commit 256edca91c8d228e828ab2b27877cd8f95f50035 at: https://605f3a8705956ab9f4bedb66--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by leecalcote** [comment]
Thank you, @Anita-ihuman. 🙂

**Pull request merged**